### ISSUE TITLE
Legend: Aligning parts of the legend text on the right side

### ISSUE THREAD

**Issue by Chnoch** [amcharts/amcharts4#361]
I'm trying to recreate the following legend:
![image](https://user-images.githubusercontent.com/247564/45616961-2d5de880-ba71-11e8-91d2-04c0f90ae211.png)

I'd like to align the currency and the value on the right side of the legend container, which has 100% width. Is it possible to only align part of the legend text on the right side with the styling options provided by amcharts / default SVG's? Or is there another way to recreate this layout?

**Comment by Chnoch** [comment]
Perfect, thanks a lot!

**Comment by sassomedia** [comment]
@darlesson Have you ever tried this with a PieChart? The example is just for the the LineChart, maybe that's different. If I follow the example, it does not have any effect, the alignment does not go to the right. Especially when I use `chart.legend.position = "right";` the whole value disappears.

**Comment by martynasma** [comment]
@sassomedia What do you mean the whole value disappears? `chart.legend.position = "right";` should work perfectly fine for PieChart just as well:

https://codepen.io/team/amcharts/pen/f2b670a89ac693e2826c2fcc0a7dad5d/

**Comment by sassomedia** [comment]
@martynasma Sorry for not specifying, in my project I use the chart with a JSON-config and the legend in a separate container. Not sure what I did wrong initially, but it does indeed work in a basic [example](https://jsfiddle.net/sassomedia/boug2zj8/) with separate container.

Following that, it still didn't work for my setup. Finally figured it out though; despite having the container already sized to 100%, I also had to specifically add `width` & `height` properties in the config file and now it's working. Thanks! 👍 
```json
legend: {
  "height": "100%",
  "parent": {},
  "position": "right",
  "width": "100%",
},
```
